api/lib/cat/answer.js

```diff
@@ -1,8 +1,16 @@
 class Answer {
   constructor(challenge, result) {
-    this.challenge = challenge;
+    this.challenge = challenge || this.emptyChallenge;
```

Comment:
J'aime bien l'idée du binaryOutcome.
Je pense que si `this.emptyChallenge` n'est pas réutilisé on peut juste appeler une méthode qui va faire la condition et ramener ce qu'il va bien. wdyt ?

Reply:
Je l'ai changé pour une function _getChallenge qui prend le challenge et fait la vérif dedans